Dockerfile

```diff
@@ -53,18 +54,18 @@ RUN curl -o /tmp/AdobeICCProfiles.zip https://download.adobe.com/pub/adobe/iccpr
 # Install princexml
 # ---------------------------
 
-ENV PRINCE_VERSION=14.2-1
-ENV PRINCE_UBUNTU_BUILD=20.04
+ENV PRINCE_VERSION=15-1
+ENV PRINCE_UBUNTU_BUILD=22.04
 
-RUN wget --directory-prefix=/tmp/ https://www.princexml.com/download/prince_${PRINCE_VERSION}_ubuntu${PRINCE_UBUNTU_BUILD}_amd64.deb
+RUN wget --directory-prefix=/tmp/ https://www.princexml.com/download/prince_${PRINCE_VERSION}_ubuntu${PRINCE_UBUNTU_BUILD}_$(dpkg --print-architecture).deb
 
-RUN gdebi --non-interactive /tmp/prince_${PRINCE_VERSION}_ubuntu${PRINCE_UBUNTU_BUILD}_amd64.deb
+RUN gdebi --non-interactive /tmp/prince_${PRINCE_VERSION}_ubuntu${PRINCE_UBUNTU_BUILD}_$(dpkg --print-architecture).deb
 
 # ---------------------------
 # Install jq and Pandoc
 # ---------------------------
 ENV JQ_VERSION='1.6'
-ENV PANDOC_VERSION='2.11.3.2'
+ENV PANDOC_VERSION='2.12'
```

Comment:
Are you sure this version of pandoc will work with the docx pipeline? I remember running into problems when I tried to use newer versions.

Reply:
I ran into this error... probably what you were referring to... hmm.

```sh
Converting to docx: 8b89d172-2927-466f-8661-01abc7ccdba4@9f5ac30:100d4b89-50fd-47f5-b530-99221cb6a5bd.xhtml
[WARNING] Could not convert TeX math '\left. \int\frac{du}{a^{2} + u^{2}} \right. = \frac{1}{a}\mspace{1mu}\text{tan}^{-1}\frac{u}{a} + C', rendering as TeX:
   \frac{1}{a}\mspace{1mu}\text{tan}^{-1}\
                     ^
  unexpected control sequence \mspace
  expecting "%", "\\label", "\\nonumber", whitespace or "\\mspace"
[WARNING] Could not convert TeX math '\left. \int\frac{du}{u\sqrt{u^{2} - a^{2}}} = \frac{1}{\left| a \right|}\mspace{1mu}\text{sec}^{-1}\frac{\left| u \right|}{a} + C \right.', rendering as TeX:
  | a \right|}\mspace{1mu}\text{sec}^{-1}\
                     ^
  unexpected control sequence \mspace
  expecting "%", "\\label", "\\nonumber", whitespace or "\\mspace"
...
Failing because there were warnings.
Failed to build actual/osbooks-calculus-bundle
```

Reply:
@TylerZeroMaster mentioned that it could be an apt package version problem... also mentioned that mspace may be in a separate package

Reply:
I was thinking about [this part](https://pandoc.org/MANUAL.html#creating-a-pdf) of the documentation. It mentions [TeX Live](https://www.tug.org/texlive/) which includes packages required by LaTeX like amsmath.